rust/auction/program/src/processor.rs

```diff
@@ -90,7 +90,7 @@ pub struct AuctionData {
     pub bid_state: BidState,
 }
 
-pub const MAX_AUCTION_DATA_EXTENDED_SIZE: usize = 8 + 9 + 2 + 200;
+pub const MAX_AUCTION_DATA_EXTENDED_SIZE: usize = 8 + 9 + 2 + 9 + 200;
```

Comment:
You need to use the existing padding, you can't make the accounts larger. So this would be + 9 + 191